specification/schemas/CombinedFiles.json

```diff
@@ -0,0 +1,85 @@
+{
+  "type": "object",
+  "allOf": [
+    {
+      "$ref": "#/components/schemas/MinimalCombinedFiles"
+    },
+    {
+      "required": [
+        "attributes",
+        "relationships"
+      ],
+      "properties": {
+        "attributes": {
+          "type": "object",
+          "additionalProperties": false,
+          "properties": {
+            "document_type": {
+              "type": "string",
+              "enum": [
+                "label"
+              ],
+              "example": "label"
+            },
+            "page_size": {
+              "type": "string",
+              "enum": [
+                "A4",
+                "A5",
+                "A6"
+              ],
+              "example": "A4"
+            },
+            "starting_position": {
+              "type": "string",
+              "enum": [
+                "top-left",
+                "top-right",
+                "bottom-left",
+                "bottom-right"
+              ],
+              "example": "top-left"
+            }
```

Comment:
You can add `"default": "label"`, `"default": "A4"` and `"default": "top-left"` to these attributes to indicate what the default values will be when not supplied.